tests/test_package.py

```diff
@@ -185,6 +185,12 @@ def test_package_indexing_truncates_unsafe_fields(self, mock_solr, app):
                     for k, v in actual.items()
                     if k in expected
                 }
+
+                # status is coming back as str type rather than json
+                # set it back to json for full comparison
+                if type(actual_common['status']) is str:
+                   actual_common['status'] = json.loads(actual_common['status'])
```

Comment:
Just checking you meant to include this. (no mention in commit messages etc. so just mentioning in case it was an accidental add)

Reply:
yes, for some reason its needed for the test set up, I'm not going to dive too much into it as other priorities at the moment